lib/pbio/drv/clock/clock_ev3.c

```diff
@@ -63,15 +62,17 @@ void systick_isr_C(void) {
  */
 void systick_suspend(void) {
     /* Disable the timer interrupt */
-    TimerDisable(SOC_TMR_0_REGS, TMR_TIMER34);
```

Comment:
I think these hooks stem from another project and we don't use them, so we could drop them.

Reply:
Removed these functions in the latest version